### ISSUE TITLE
Power Browser Leveraging Existing Browser

### ISSUE THREAD

**Issue by cretz** [cretz/software-ideas#102]
* Use Chrome in "app" mode or FF in kiosk mode or something that doesn't have tab bar
* Have extension that can communicate w/ host daemon on what tabs and what can be opened
* Make tree tab control outside of browser that controls it
  * Would like to avoid Qt if I can (don't like QML or C++ or Python)
  * WinForms? Ug, state f cross-platform non-C/C++ UI libraries is very weak
* Consider blank area for browser that moves the browser around to it constantly...w/ win32 I can re-parent the hwnd quite likely
* Basically I want an easier-to-maintain #46 ...I'm tired of constantly updating CEF
* Ideally could abstract the browser engine to a common iface so could have choice (or multiple) at once

**Comment by cretz** [comment]
Ooh, have a look at https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2 if we're willing to lose cross platform (beta [just announced](https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2))

**Comment by ghost** [comment]
Is this idea about creating a custom shell over an existing browser? Firefox used to support this kind of skinning but they dropped embedding support. Chromium is embeddable AFAIK.

**Comment by cretz** [comment]
Started toying with this idea at https://github.com/cretz/ffembedpoc
